### ISSUE TITLE
Developer Documentation

### ISSUE THREAD

**Issue by cfi2017** [SlimeKnights/TinkersToolLeveling#109]
How does one go about setting up a working developing environment for this mod? 

Given a working system with Java 8, git and (after trying to find out what the scala import in the gradle build conf is for) even sbt installed, how does one actually get this to work?

- a simple gradle build won't work, unable to resolve class `scala.tools.nsc.io.Jar`
- `sbt` does... something, but compile etc. fails due to missing dependencies
- IntelliJ doesn't import the project due to the gradle wrapper being out of date
- gradle wrapper doesn't update due to not being able to resolve `scala.tools.nsc.io.Jar`

The current released version of the mod is 1.1.0, however the gradle properties file lists an old version.

**Comment by bonii-xx** [comment]
You're right, apparently I still had some changes locally. That should fix the scala thing.

Other than that you can set it up like any minecraft mod for 1.12, using `gradle setupDecompWorkspace idea`
